roles/edpm_libvirt/tasks/virsh-secret.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Create XML file for virsh secret on container host
+  ansible.builtin.template:
+    src: libvirt_virtqemud/secret.xml.j2
+    dest: /tmp/secret.xml
+    mode: "0600"
+
+- name: Copy XML file into libvirt_virtqemud container
+  ansible.builtin.command: podman cp /tmp/secret.xml libvirt_virtqemud:/tmp/secret.xml
+
+- name: Delete XML secret file on host
+  ansible.builtin.file:
+    path: /tmp/secret.xml
+    state: absent
+
+- name: Run virsh secret-define inside the libvirt_virtqemud container
+  changed_when: true
+  ansible.builtin.shell: "podman exec libvirt_virtqemud bash -c '{{ '; '.join(cmds) }}'"
```

Comment:
ack makes sense

you might have been able to fix that via changing 

"podman exec libvirt_virtqemud bash -c '{{ '; '.join(cmds) }}'"
to
"podman exec libvirt_virtqemud bash -c \"{{ '; '.join(cmds) }}\""
e.g. it may just be an issue with the nesting of _'_
but I'm fine with shell just checking you had a reason not to use command in this case.